src/util/merkleblock.rs

```diff
@@ -500,7 +500,7 @@ mod tests {
 
     use bitcoin_hashes::hex::{FromHex, ToHex};
     use bitcoin_hashes::{sha256d, Hash};
-    use secp256k1::rand::{weak_rng, Rng, XorShiftRng};
+    use secp256k1::rand::prelude::*;
```

Comment:
Feel free to ignore this change, but I think it'd be preferable to explicitly `use` instead of using `prelude::*` for clarity